### PR TITLE
support to update tx hash

### PR DESCRIPTION
**Pull request by boern**

### CODE REVIEW

File: hub/src/state.rs [hunk 1]
```diff
@@ -47,6 +47,8 @@ pub struct HubState {
     pub token_position: StableBTreeMap<TokenKey, Amount, Memory>,
     #[serde(skip, default = "memory::init_ledger")]
     pub cross_ledger: StableBTreeMap<TicketId, Ticket, Memory>,
+    #[serde(skip, default = "memory::init_tx_hashes")]
```
Comment (shenao78) [inline hub/src/state.rs:50]: skip_ skip_deserializing?

File: hub/src/state.rs [hunk 2]
```diff
@@ -781,6 +797,12 @@ impl HubState {
         Ok(tickets)
     }
 
+    pub fn qet_tx_hash(&self, ticket_id: &TicketId) -> Result<TxHash, Error> {
```
Comment (shenao78) [inline hub/src/state.rs:800]: `get_tx_hash`

File: scripts/hub_test.sh [hunk 3]
```diff
@@ -170,6 +170,11 @@ dfx canister call omnity_hub query_tickets '(opt "Ethereum",0:nat64,5:nat64)'
 dfx canister call omnity_hub get_chain_tokens '(opt "Optimistic",null,0:nat64,5:nat64)'
 dfx canister call omnity_hub get_chain_tokens '(opt "Starknet",null,0:nat64,5:nat64)'
 
+# update tx hash
+dfx canister call omnity_hub update_tx_hash '("f8aee1cc-db7a-40hea-80c2-4cf5e6c84c21","f8aee1cc-db7a-40hea-80c2-4cf5e6c84c21")'
```
Comment (TingALin) [inline scripts/hub_test.sh:174]: tx_hash format might be something like "0x33766543beaf0e16c764fcd553b332f38c86865f88da74bf0ee9e453a4bdd910" ?

Comment (boern) [reply scripts/hub_test.sh:174]: just test data

**Review by shenao78 — APPROVED** [review]

**Review by TingALin — APPROVED** [review]

**Pull request merged**